pydocx/parsers/Docx2Html.py

```diff
@@ -32,6 +32,14 @@ def parsed(self):
         )
         return content
 
+    @property
+    def parsed_without_head(self):
+        content = super(Docx2Html, self).parsed
+        content = "<html><body>{body}</body></html>".format(
+            body=content,
+        )
+        return content
```

Comment:
It sounds perfectly acceptable to me. I just thought that a fairly common recurring event in use cases for PyDocX might be displaying the parsed output in a web page, and if so an additional property to make that simpler for the end user could be worthwhile. 

I'll update this early tomorrow, after making the changes on my side of the code. Thanks for the advice :+1: